tests/dev/script-user-defined-pk-parameters.R

```diff
@@ -0,0 +1,30 @@
+rm(list = ls())
+library(ospsuite)
+library(ospsuite.reportingengine)
+
+# One time initialization of PK Parameters
+ospsuite::updatePKParameter("C_max", displayName = "C max")

```

Comment:
There is an issue here with settings the unit to a mass unit in this case. I have to come up with something that will allow the user to do this